docs/app/pages/components/components-sections/timeline/timeline/timeline.component.ts

```diff
@@ -0,0 +1,84 @@
+import { Component } from '@angular/core';
+import { DocumentationSectionComponent } from '../../../../../decorators/documentation-section-component';
+import { BaseDocumentationSection } from '../../../../../components/base-documentation-section/base-documentation-section';
+import 'chance';
+
+@Component({
+    selector: 'uxd-components-timeline',
+    templateUrl: './timeline.component.html'
+})
+@DocumentationSectionComponent('ComponentsTimelineComponent')
+export class ComponentsTimelineComponent extends BaseDocumentationSection {
+
+    constructor() {
+        super(require.context('./snippets/', false, /\.(html|css|js|ts)$/));
+    }
+
+    _developer = chance.name();
+
+    tickets = [{
+        color: 'critical',
+        shortDate: 'Mon 29th',
+        title: 'Addition of Timeline component',
+        date: new Date(2018, 0, 29, 11, 10),
+        url: '#',
+        id: '1234',
+        status: 'failed by',
+        assignee: chance.name()
+    }, {
+        color: 'alternate3',
+        shortDate: 'Fri 26th',
+        title: 'Addition of Timeline component',
+        date: new Date(2018, 0, 26, 14, 0),
+        url: '#',
+        id: '1234',
+        status: 'reviewed by',
+        assignee: chance.name()
+    }, {
+        color: 'alternate1',
+        shortDate: 'Wed 24th',
+        title: 'Addition of Timeline component',
+        date: new Date(2018, 0, 24, 9, 20),
+        url: '#',
+        id: '1234',
+        status: 'assigned for development to',
+        assignee: this._developer
+    }, {
+        color: 'accent',
+        shortDate: 'Mon 22nd',
+        title: 'Addition of Timeline component',
+        date: new Date(2018, 0, 22, 13, 45),
+        url: '#',
+        id: '1234',
+        status: 'recorded by',
+        assignee: chance.name()
+    }];
+
+    addEvent = function () {
```

Comment:
Remove `= function`. TypeScript class functions should be written like:

```typescript
addEvent(): void {
  ...
}
```

Reply:
Using correct syntax for class function.